### PR TITLE
[611] feat: redesign of directory page

### PR DESCRIPTION
**Pull request by Sperthix**
**Changes proposed in this Pull Request**
Redesigned directory homepage.

**Testing instructions**
Go to [this](http://postaffiliatepro.local/affiliate-program-directory/) page
Check the design

**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

Close QualityUnit/web-issues#611

### CODE REVIEW

File: assets/styles/pages/Archive.scss [hunk 1]
```diff
@@ -198,41 +214,41 @@
...
+					line-clamp: 3;
+					-webkit-box-orient: vertical;
 
 					/* for better support */
-					display: block;/* or inline-block */
-					text-overflow: ellipsis;
-					word-wrap: break-word;
-					overflow: hidden;
-					max-height: 60px;
+					// display: block;
```
Comment (jremen) [inline assets/styles/pages/Archive.scss:234]: Remove commented out code

File: assets/styles/pages/Archive.scss [hunk 2]
```diff
@@ -288,13 +317,13 @@
...
-			font-size: 16px;
+			font-size: 1rem;
 			line-height: 48px;
 			margin-right: 20px;
 		}
 
 		select {
-			font-size: 16px;
+			font-size: 1rem;
 			line-height: 48px;
```
Comment (jremen) [inline assets/styles/pages/Archive.scss:327]: use line-height: 3 as 1rem = 16px * 3 = 48px

File: assets/styles/pages/Archive.scss [hunk 3]
```diff
@@ -288,13 +317,13 @@
 		}
 
 		label {
-			font-size: 16px;
+			font-size: 1rem;
 			line-height: 48px;
```
Comment (jremen) [inline assets/styles/pages/Archive.scss:321]: line-height: 3

File: assets/styles/pages/Archive.scss [hunk 4]
```diff
@@ -315,7 +344,7 @@
 			margin-bottom: 0;
 
 			li {
-				font-size: 16px;
+				font-size: 1rem;
 				line-height: 28px;
```
Comment (jremen) [inline assets/styles/pages/Archive.scss:348]: line-height: 1.75

**Review by jremen — CHANGES_REQUESTED** [review]
Fix obvious visual bugs and fix line-height: to whole number instead of pixels (divide current px value by font size used)

Screenshot 2022-10-27 at 09 43 20
-images.githubusercontent.com/7055428/198222390-2edf6060-2a0e-4460-849b-bed7418db04c.png)

**Comment by mvstepanek** [comment]
@jremen, make those item cards clickable and make padding bigger in section Affiliate program categories.

**Comment by jremen** [comment]
> @jremen, make those item cards clickable and make padding bigger in section Affiliate program categories.

Done

**Review by mvstepanek — APPROVED** [review]

**Pull request merged**